docs/structure-of-a-contract.rst

```diff
@@ -42,6 +42,41 @@ Functions are the executable units of code within a contract.
 and have different levels of visibility (:ref:`visibility-and-getters`)
 towards other contracts. Functions must be decorated with either @public or @private.
 
+Default function
+----------------
+
+A contract can also have a default function, which is executed on a call to the contract if no other functions match the given function identifier (or if none was supplied at all, such as through someone sending it Eth). It is the same construct as fallback functions `in Solidity <https://solidity.readthedocs.io/en/latest/contracts.html?highlight=fallback#fallback-function>`_. 
+
+This function is always named `__default__` and must be annotated with `@public` and `@payable`. It cannot have arguments and cannot return anything.
+
+Furthermore, this function is executed whenever the contract receives plain Ether (without data). This is why the default function cannot accept arguments and return values - it is a design decision of Ethereum to make no differentiation between sending ether to a contract or a user address.
+
+**Example:**
+
+::
+
+    Sent: event({sender: indexed(address)})
+    
+    @public
+    @payable
+    def __default__():
+        log.Sent(msg.sender)
+
+
+Considerations
+~~~~~~~~~~~~~~
+
+Unlike Solidity, Vyper generates a default function is one isn't found, in the form of a REVERT call. Note that this still `generates an exception <https://github.com/ethereum/wiki/wiki/Subtleties>`_ and thus will not succeed in receiving funds. 
+
+Send calls to the contract give it a maximum stipend of 2300 gas, leaving not much room to perform other operations except basic logging. The following operations will consume more gas than the 2300 gas stipend:
```

Comment:
You can still choose to include a higher gas amount when you send ETH to an address, which allows more complex fallback functions that will automatically fail unless extra gas is included. At least thats how it is with Solidity fallbacks, hoping its the same here.

Many ICOs allowed purchasing by sending ETH to the fallback and including ~200,000 gas. I have even heard of MEW automatically s the higher gas amount when it detected a user sending to a known ICO with a fallback.

Reply:
@haydenadams yes it is I believe, created an issue to demonstrate this as a tests. 
@liamzebedee   please make it clear that one can still send more gas to a default function, if you don't use `send`. for example raw_call / CALL allows you to send as much gas as you like.